tests/Users/Controllers/UserControllerTest.php

```diff
@@ -191,4 +191,113 @@ public function testViewUser()
         $this->assertEquals($user->website, $results->entities[0]->website);
 
     }
+
+    public function testBrowseNotFound()
+    {
+        $res = $this->call('GET', '/users');
+        $this->assertEquals(200, $res->getStatusCode());
+        $results = json_decode($res->getContent());
+        $this->assertEquals(1, count($results->entities));
```

Comment:
@gghoantt test browse not found mà chỗ ni count bằng 1. nhìn vào thấy ko hợp lý á em.
Anh đề xuất chỗ này em gọi hàm remove hết user để test